CBLClient/Apps/CBLTestServer-iOS/CBLTestServer-iOS/Server/PeerToPeerRequestHandler.swift

```diff
@@ -95,10 +169,30 @@ public class PeerToPeerRequestHandler {
                 replicatorConfig = ReplicatorConfiguration(database: database, target: endpoint)
             }
             
+            if auth != nil {
+               replicatorConfig.authenticator = auth
+            }
+            if serverVerificationMode != false {
+                replicatorConfig.acceptOnlySelfSignedServerCertificate = true
+            }
+            
+            if tlsAuthType == "self_signed" {
+                let certData = try! dataFromResource(name: "identity/certs", ofType: "p12")
+                try! TLSIdentity.deleteIdentity(withLabel: "CBL-Cert")
+                let identity = try! TLSIdentity.importIdentity(withData: certData, password: "123", label: "CBL-Cert")
+                replicatorConfig.pinnedServerCertificate = identity.certs[0]
+                print("pinned the certs to Replicator >>>>>>>>>>>>>>>")
+            }
+            if tlsAuthenticator != false {
+                let clientCertData = try dataFromResource(name: "identity/client", ofType: "p12")
+                try! TLSIdentity.deleteIdentity(withLabel: "CBL-Cert")
+                let identity = try TLSIdentity.importIdentity(withData: clientCertData, password: "123", label: "CBL-Cert")
+                replicatorConfig.authenticator = ClientCertificateAuthenticator(identity: identity)
+                print("Added Autheticator to Replicator >>>>>>>>>>>>>>>")
```

Comment:
when I saw >>>>>>>> I thought your code has git conflicts :) is that possible to use other sign, like ======= or something else?